src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -82,6 +82,40 @@ std::string postUpdateContext
     return answer;
   }
 
+  //
+  // If upcr.contextElementResponseVector contains >= TWO responses and the second of them is a 'SccFound',
+  // then skip the first, the one that isn't 'SccFound'.
+  // Also, if the first of them is 472, then skip the second
+  //
+  // FIXME P6: Temporary hack for 'Entity found but Attribute not found':
+  //           See FIXME P6 'Temporary hack ...' 
+  //           in MongoCommonUpdate.cpp, function processContextElement
+  //
+
```

Comment:
If I'm understanding the hack correctly, the "false double" situation happens when why is "Attribute Not Found" and the operation is updateContext. In this case, we have an SccInvalidParamter and a SccFound. I undestand that this circustance is "captured" by the first `if`, but... why we need the second `if`?

Moreover, I think you are assuming implicitely that the updateContext request have 1 CE. However, the updateContext in general could have N CEs. Could be some case of wrong behaviour in the case of the client uses updateContext with 2 element? E.g. in the case of updateContext with 2 elements, the first one resulting in SccInvalidParamer (but not of type of "Attribute not found", e.g. "location nature of an attribute has to be defined at creation time, with APPEND") and the second one going SccOk, the second if will wroning remove the SccOk, thus the client is sending 2 elements but receiving 1 response, which is inconsistent. This breaks existing functionality, not related with CPr forwarding...

Maybe a solution would be something like this:

// This check is to avoid wrongly removing responses when number of CE in the request is greater than 1
if (&parseDataP->upcr.res.contextElementVector.size() == 1)
{
  if (upcr.contextElementResponseVector.size() >= 2)
  {
  ...
  }
  if (upcr.contextElementResponseVector.size() >= 2)
  {
  ...
  }
}

Probably there would be smarter solutions, but I'm not able to find better ones right now...